Makefile

```diff
@@ -0,0 +1,19 @@
+RUNTIME_PACKAGE = runtime.zip
```

Comment:
Let's introduce the make file in another PR and just have this PR relate to error handling :-)